lib/wasix/src/runtime/resolver/wapm_source.rs

```diff
@@ -343,15 +343,36 @@ fn decode_summary(
 ) -> Result<PackageSummary, Error> {
     let WapmWebQueryGetPackageVersion {
         manifest,
-        distribution:
+        v2:
             WapmWebQueryGetPackageVersionDistribution {
-                webc_version,
-                pirita_sha256_hash,
-                pirita_download_url,
+                webc_version: v2_webc_version,
+                pirita_sha256_hash: v2_pirita_sha256_hash,
+                pirita_download_url: v2_pirita_download_url,
+            },
+        v3:
+            WapmWebQueryGetPackageVersionDistribution {
+                webc_version: v3_webc_version,
+                pirita_sha256_hash: v3_pirita_sha256_hash,
+                pirita_download_url: v3_pirita_download_url,
             },
         ..
     } = pkg_version;
 
+    let (webc_version, pirita_sha256_hash, pirita_download_url) =
+        if std::env::var("WASMER_USE_WEBCV3").is_ok() {
```

Comment:
This needs to be a configuration variable that is passes to WASIX. WASIX should NOT access env vars, only the CLI should